source/contracts/trading/Cash.sol

```diff
@@ -47,6 +47,17 @@ contract Cash is DelegationTarget, ITyped, VariableSupplyToken, ICash {
         return true;
     }
 
+    function withdrawEtherToIfPossible(address _to, uint256 _amount) external returns (bool) {
+        require(_amount > 0 && _amount <= balances[msg.sender]);
+        if (_to.send(_amount)) {
```

Comment:
This change generally makes me a little uneasy, given that we are attempting to send value twice. It does make me feel a tad better to think about `.transfer()` as `require(.send())`, so if someone could influence the boolean response while still accepting the funds, there would be consequences everywhere.